packages/components/src/components/scrollable/scrollable.stories.ts

```diff
@@ -0,0 +1,166 @@
+import '../../solid-components';
+import { html } from 'lit';
+import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
+import { waitUntil } from '@open-wc/testing-helpers';
+import { withActions } from '@storybook/addon-actions/decorator';
+
+const { argTypes, parameters } = storybookDefaults('sd-scrollable');
+const { overrideArgs } = storybookHelpers('sd-scrollable');
+const { generateTemplate } = storybookTemplate('sd-scrollable');
+
+const defaultSlotContent = `
+  <div class="slot slot--border slot--text items-start" style="height:max-content; width:max-content; padding: 1rem; justify-content:start;">
+    <p>Scroll and give it a try!</p>
+    <br/>
+    <p>This is a long scrollable content.</p>
+    <p>It contains multiple paragraphs and lines.</p>
+    <p>The content is intentionally long to trigger scrolling. You can scroll horizontally and vertically.</p>
+    <p>The scrollable component will display shadows and buttons based on the props.</p>
+    <p>Customize the content and attributes as needed.</p>
+  </div>
+`;
+
+export default {
+  title: 'Components/sd-scrollable',
+  component: 'sd-scrollable',
+  args: overrideArgs({
+    type: 'slot',
+    name: 'default',
+    value: defaultSlotContent
+  }),
+  argTypes,
+  parameters: { ...parameters },
```

Comment:
What is this?
![CleanShot 2024-05-14 at 21 20 30@2x](https://github.com/solid-design-system/solid/assets/26542182/10318cf5-30ca-40a1-b2d6-c96ce3d8b95c)

Just saw it in every story.

Reply:
Got it!